Leetcode/aoc_2015_d6.py

```diff
@@ -38,6 +38,36 @@
 with open("aoc_2015_d6_input.txt") as file:
 	instructions = [line.rstrip("\n") for line in file]
 
+
+def light_on(grid: list[list], r1: int, r2: int, c1: int, c2: int):
+	for row in range(r1, r2 + 1):
+		for col in range(c1, c2 + 1):
+			grid[row][col] = True
+
+
+def light_off(grid: list[list], r1: int, r2: int, c1: int, c2: int):
+	for row in range(r1, r2 + 1):
+		for col in range(c1, c2 + 1):
+			grid[row][col] = False
```

Comment:
This is fine, but a `set_light(..., state: bool)` can replace these two functions. Just set the desired on/off state as a param.

Reply:
Makes sense. Done.